tpu-client/src/nonblocking/tpu_client.rs

```diff
@@ -233,12 +233,10 @@ impl LeaderTpuCache {
             }
         }
 
-        if let Some(Ok(epoch_info)) = cache_update_info.maybe_epoch_info {
-            self.slots_in_epoch = epoch_info.slots_in_epoch;
-            self.epoch_slot_boundary = epoch_info
-                .absolute_slot
-                .saturating_sub(epoch_info.slot_index)
-                .saturating_add(epoch_info.slots_in_epoch);
+        if let Some(Ok(epoch_schedule)) = cache_update_info.maybe_epoch_schedule {
```

Comment:
hmm... problem for a future pr, but i'm guessing eating any client errors from updating `cache_update_info` isn't exactly a helpful property